org.opentosca.container.api/src/org/opentosca/container/api/dto/situations/SituationTriggerDTO.java

```diff
@@ -17,6 +17,8 @@
 import com.google.common.collect.Lists;
 import com.google.common.collect.Sets;
 
+import io.swagger.annotations.ApiModelProperty;
```

Comment:
interesting introduction here, shouldn't this be a PR of it's own right, adding swagger annotations to all existing DTOs?